simple_salesforce/api.py

```diff
@@ -33,7 +33,7 @@ class Salesforce(object):
     # pylint: disable=too-many-arguments
     def __init__(
             self, username=None, password=None, security_token=None,
-            session_id=None, instance=None, instance_url=None,
+            session_id=None, instance=None, instance_url=None, custom_url=None,
```

Comment:
`custom_url` should be added as the last parameter to preserve backwards compatibility. Ordering can be changed in a major release.